### PR TITLE
Systemd and exhibitor integration

### PR DESCRIPTION
**Pull request by alex-roman**

### CODE REVIEW

File: manifests/service.pp [hunk 1]
```diff
@@ -42,20 +42,32 @@
         }
     }
 
+  if $zookeper::config::exhibitor_managed == 'false' {
```
Comment (travees) [inline manifests/service.pp:45]: This isn't necessary. If exhibitor is managing the zookeeper service then none of the resources in this class should be used. It's probably better to modify the `if` statement in init.pp (https://github.com/opentable/puppet-zookeeper/blob/master/manifests/init.pp#L158) to be:
`if ($manage_service) and (! $exhibitor_managed) {`

Comment (maximedevalland) [reply manifests/service.pp:45]: @travees @alex-roman : are we good to go with this?

Comment (alex-roman) [reply manifests/service.pp:45]: @MaximeDevalland @travees sure!

Comment (travees) [reply manifests/service.pp:45]: I have a puppet-modules PR related to this. 
https://github.com/opentable/puppet-modules/pull/14625

**Review by travees — APPROVED** [review]

**Pull request merged**